### PR TITLE
[wait for #1782] [modelloader] Support learning rate scheduler

### PR DESCRIPTION
**Pull request by kparichay**
Support learning rate scheduler with model loader along with a unittest.

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

File: nntrainer/models/model_loader.cpp [hunk 1]
```diff
@@ -68,6 +99,7 @@ int ModelLoader::loadOptimizerConfigIni(dictionary *ini, NeuralNetwork &model) {
     std::shared_ptr<ml::train::Optimizer> optimizer =
       createOptimizerWrapped(opt_type, properties);
     model.setOptimizer(optimizer);
+    loadLearningRateSchedulerConfigIni(ini, optimizer);
```
Comment (zhoonit) [inline nntrainer/models/model_loader.cpp:102]: what if learning rate scheduler exist but optimizer does not exist?

File: test/ccapi/unittest_ccapi.cpp [hunk 2]
```diff
@@ -196,7 +196,7 @@ static nntrainer::IniSection outputlayer("outputlayer",
  */
 TEST(nntrainer_ccapi, train_with_config_01_p) {
   std::unique_ptr<ml::train::Model> model;
-  ScopedIni s("test_train_01_p",
+  ScopedIni s("ccapi_test_train_01_p",
```
Comment (zhoonit) [inline test/ccapi/unittest_ccapi.cpp:199]: nice finding :)

File: nntrainer/models/model_loader.cpp (hunk 1 above)
Comment (kparichay) [reply nntrainer/models/model_loader.cpp:102]: Then it will have no effect as the learning rate scheduler is used only by the optimizer.

Comment (zhoonit) [reply nntrainer/models/model_loader.cpp:102]: I think it's reasonable. Could we issue a warning for this case?

If someone add sch in the ini file and add optimizer with model api, they might expect sch is added.

**Review by zhoonit — COMMENTED** [review]
Oh and could you update ini saving as well? since we don't have a test for this, physically checking if learning rate section is created will be more than enough.

File: nntrainer/models/model_loader.cpp (hunk 1 above)
Comment (kparichay) [reply nntrainer/models/model_loader.cpp:102]: There is an existing warning: `There is no [Optimizer] section in the given ini file. This model can only be used for inference`. Do you want one more warning?

Comment (zhoonit) [reply nntrainer/models/model_loader.cpp:102]: I see. Maybe adding `[Learning Schedule] section will be ignored if any` their would work. Thanks :)

**Review by lhs8928 — APPROVED** [review]
LGTM

**Comment by jijoongmoon** [comment]
This pr is included in #1871
